README.md

```diff
@@ -63,11 +61,11 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Vet Clinic] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
```

Comment:
- [ ] Please delete the template placeholder texts still present in some sections of your readme, this will make your readme more professional

Reply:
please, can you explain

Reply:
HI @Outkarm, please delete `> Describe your project in 1 or 2 sentences. `  add your description, it would be better to take your time and make your readme descriptive 👍🏿